src/OrderFormAcceptanceTests.Steps/Steps/CatalogueSolutions.cs

```diff
@@ -313,22 +326,32 @@ public void ThenTheSaveButtonIsEnabled()
         [Given(@"the User is presented with the Catalogue Solution edit form for a variable flat price")]
         public void GivenTheUserIsPresentedWithTheCatalogueSolutionEditFormVariableFlatPrice()
         {
+            CommonSteps common = new(Test, Context);
```

Comment:
I like the `new()` syntax in many places but I think `var common = new CommonSteps(Test, Context);` just reads a bit better in this context. However, if you're happy with/prefer your existing version then that's fine with me.

(Same below.)